src/PhpParser/Printer/BetterStandardPrinter.php

```diff
@@ -153,6 +154,14 @@ public function pFileWithoutNamespace(FileWithoutNamespace $fileWithoutNamespace
         return ltrim($content);
     }
 
+    /**
+     * required since PHPStan 1.10.20
+     */
+    protected function pPHPStan_Node_AlwaysRememberedExpr(AlwaysRememberedExpr $alwaysRememberedExpr): string
+    {
+        return $this->p($alwaysRememberedExpr->getExpr());
```

Comment:
@ondrejmirtes the original `sprintf()` cause addition of string:

```diff
-        $statement = match ($this->lexer->lookahead['type']) {
+        $statement = match (__phpstanRembered($this->lexer->lookahead['type'])) {
```

so I override it: